clients/linux/src/account.rs

```diff
@@ -361,11 +364,127 @@ impl Editor {
         textarea.set_left_margin(4);
         textarea.set_tab_width(4);
 
+        textarea.connect_paste_clipboard(Self::on_paste_clipboard(m, c, lbs));
+        textarea.connect_button_press_event(Self::on_button_press(m));
+
         let textview = textarea.upcast_ref::<gtk::TextView>();
+
         let gspell_view = gspell::TextView::get_from_gtk_text_view(textview).unwrap();
         gspell_view.basic_setup();
 
-        textarea.connect_button_press_event(closure!(m => move |w, evt| {
+        let scroll = GtkScrolledWindow::new(None::<&GtkAdjustment>, None::<&GtkAdjustment>);
+        scroll.add(&textarea);
+
+        let cntr = GtkStack::new();
+        cntr.add_named(&empty, "empty");
+        cntr.add_named(&info, "folderinfo");
+        cntr.add_named(&scroll, "scroll");
+
+        let highlighter = LanguageManager::get_default().unwrap_or_default();
+        let lang_paths = highlighter.get_search_path();
+
+        let mut str_paths: Vec<&str> = lang_paths.iter().map(|path| path.as_str()).collect();
+        let lang_specs = get_language_specs_dir();
+        str_paths.push(lang_specs.as_str());
+        highlighter.set_search_path(str_paths.as_slice());
+
+        Self {
+            info,
+            textarea,
+            highlighter,
+            change_sig_id: RefCell::new(None),
+            cntr,
+            messenger: m.clone(),
+        }
+    }
+
+    fn on_paste_clipboard(
+        m: &Messenger,
+        c: &Arc<LbCore>,
+        lbs: &Rc<RefCell<LbState>>,
```

Comment:
I tried to organize the code base so that `LbState` would just be manipulated within `LbApp`, and I think so far that has been maintained. This scenario isn't a good reason to change that imo. Instead, I'd adhere to that principle and pass a Message back to LbApp.